recipes/libignition-transport/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+mkdir build
+cd build
+
+cmake .. \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_PREFIX_PATH=$PREFIX -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_INSTALL_LIBDIR=lib \
+      -DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=True \
+      -DBUILD_TESTING=False
```

Comment:
Why skip tests?

Reply:
I do that to save the build time.

Reply:
You should build tests and run them to make sure the package is working as expected (unless the tests take a very long time)

Reply:
Added the tests back to the recipes.